src/core/load-fragment-gperf.gperf.m4

```diff
@@ -194,6 +194,7 @@ Unit.OnFailureIsolate,           config_parse_job_mode_isolate,      0,
 Unit.IgnoreOnIsolate,            config_parse_bool,                  0,                             offsetof(Unit, ignore_on_isolate)
 Unit.IgnoreOnSnapshot,           config_parse_warn_compat,           DISABLED_LEGACY,               0
 Unit.JobTimeoutSec,              config_parse_sec_fix_0,             0,                             offsetof(Unit, job_timeout)
+Unit.JobRunningTimeoutSec,       config_parse_sec_fix_0,             0,                             offsetof(Unit, job_running_timeout)
```

Comment:
please don't use "fix_0" here. That's just a hacked version we keep around for compat with old systemd versions

(in old systemd versions we permitted setting timeouts to "0" for disabling them, but since normalized that so that only a timeout of infinity turns off timeouts. For new settings we don't need this sort of compat however, hence please use normal config_parse_sec() here)